src/rsz/src/Resizer.cc

```diff
@@ -331,8 +331,9 @@ Resizer::bufferConnectedToPorts(Instance *buffer)
   Net *in_net = db_network_->net(in_pin);
   Net *out_net = db_network_->net(out_pin);
   bool in_net_ports = hasPort(in_net);
+  bool in_net_clk = sta_->isClock(in_net);
   bool out_net_ports = hasPort(out_net);
-  return in_net_ports || out_net_ports;
+  return (in_net_ports && !in_net_clk) || out_net_ports;
```

Comment:
once again you have changed the function and left the function name inconsistent.

Did you check the failures with just the merge and not this change?
I think some of the recent resizer changes may be relevant.

Reply:
Yes, tests pass after merge. I have changed function name back.